problem-spring-web-autoconfigure/src/test/java/org/zalando/problem/spring/web/autoconfigure/security/SecurityConfigurationTest.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.problem.spring.web.autoconfigure.security;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+
+@SpringBootTest("spring.main.web-application-type=none")
+final class SecurityConfigurationTest {
+
+    @Test
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/whiskeysierra/problem-spring-web/pullRequest?prid=4540409)

Reply:
Can be ignored